demo/src/app/pages/demo/components/advanced-example-types.component.ts

```diff
@@ -14,77 +15,88 @@ export class AdvancedExamplesTypesComponent {
       name: "Leanne Graham",
       username: "Bret",
       email: "Sincere@april.biz",
-      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu."
+      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu.",
+      custom: '10|20'
```

Comment:
I think we may need a _custom renderer/viewer_ if the value is not `string`, correct? Should I consider doing this?